here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PostgresStorage.java

```diff
@@ -450,6 +439,50 @@ synchronized void initStorage(@NotNull PsqlStorage.Params params, @NotNull IoHel
     }
   }
 
+  private void initStoragePlv8(
+      PsqlStorage.@NotNull Params params, @NotNull IoHelp ioHelp, Statement stmt, PsqlConnection conn)
+      throws SQLException {
+    final Plv8Env plv8Env = new Plv8Env();
+    plv8Env.install(conn, latest.toLong());
+    conn.commit();
+  }
```

Comment:
In `plv8` setup variant we don't consider `pg_hint_plan` and `pg_stat_statements` extensions - is that intended?

Reply:
Question for @xeus2001 and @cyberhead-pl, but I will assume no for this PR.

Reply:
I'm fine with that assumption 😄